R/configuration-plan.R

```diff
@@ -160,8 +160,7 @@ ConfigurationPlan <- R6::R6Class(
           # With recursive=TRUE, overwrite images
           file.copy(
             from = file.path(inputContentSubFolder, fileName),
-            to = file.path(outputContentSubFolder, fileName),
-            recursive = TRUE
+            to = file.path(outputContentSubFolder, fileName)
```

Comment:
If my input ContentFolder looks like this:
```
Content
  -- file1.md
  -- images/
      -- pic1.png
      -- images2/
          -- pic2.png
```

Will it be copied correctly into the output content folder with this change?

Reply:
Does not work... the message is not there anymore but `file.copy` does not copy the sub files/folders

Reply:
Well, then improve pls